### DISCUSSION TITLE
How to display cron workflows in default workflows tab?

### DISCUSSION THREAD

**Discussion by kolyadin** [argoproj/argo-workflows#7735] [Q&A]
Is it possible to display cron workflows in default workflows?
For some reason i see cron workflows only in archive.

**Comment by terrytangyuan** [comment]
There's a dedicated tab for Cron Workflows. It looks like this <img width="35" alt="image" src="https://user-images.githubusercontent.com/4269898/152367579-dcc20bb7-f93e-415e-af58-96f02edfa19e.png">.

**Reply by kolyadin** [reply]
In cron workflows i can only see cron templates, and real workflows generated from cron workflows only visible in archive workflows, and i want to see those in default workflows tab.

**Reply by terrytangyuan** [reply]
I don't think we are archiving the workflows generated from corn workflows. Do you have any other archiving process in place? Could you paste some screenshots?
